src/routes/user.routes.js

```diff
@@ -10,4 +10,10 @@ router.post(`${paths.user}`, controllers.createUser);
 //PATCH /api/v1/user/:_id
 router.patch(`${paths.user}/:id`, controllers.updateUser);
 
+//GET /api/v1/user/:_id
```

Comment:
nitpick: this should be //GET /api/v1/user/:id, I don't think it affects anything, the actual path is correct